io.github.endless_sky.endless_sky.desktop

```diff
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Version=1.5
 Name=Endless Sky
+Name[hu]=Végtelen égbolt
```

Comment:
I don't know Hungarian, but I think the game name should stay the same in every language; the other languages we have here change only the description.

Reply:
There is no problem with program names being translated; however, updated so can you please merge. Thank you